tests/unit-tests/tax/tax.php

```diff
@@ -621,8 +631,144 @@ public function test__update_tax_rate_cities() {
 
 		WC_Tax::_update_tax_rate_cities( $tax_rate_id, $to_save );
 
-		$results = $wpdb->get_col( $wpdb->prepare( "SELECT location_code FROM {$wpdb->prefix}woocommerce_tax_rate_locations WHERE tax_rate_id = %d ORDER BY location_code ASC", $tax_rate_id ) );
+		$results = $wpdb->get_col(
+			$wpdb->prepare(
+				"SELECT location_code FROM {$wpdb->prefix}woocommerce_tax_rate_locations WHERE tax_rate_id = %d ORDER BY location_code ASC",
+				$tax_rate_id
+			)
+		);
 
 		$this->assertEquals( array( 'SOMEWHERE', 'SOMEWHERE_ELSE' ), $results );
 	}
+
+	public function test__price_to_display() {
```

Comment:
Please move it to `tests/unit-tests/product/functions.php`, and rename to `test_wc_get_price_to_display()`